src/popup/components/Modals/AccountCreate.vue

```diff
@@ -96,11 +92,15 @@ export default defineComponent({
 @use '../../../styles/typography';
 
 .account-create {
+  .content-wrapper {
+    padding-inline: 16px;
```

Comment:
Isn't it our `--screen-padding-x`?

Reply:
this variable doesn't work in this scope